ionic-angular/official/super/src/models/item.ts

```diff
@@ -12,8 +12,13 @@ export class Item {
   constructor(fields: any) {
     // Quick and dirty extend/assign fields to this model
     for (const f in fields) {
+      // @ts-ignore
```

Comment:
Why is this needed? The `Item` type should merge class and interface and allow an index signature.

Reply:
It's unnecessary. At first i want to suppress the signature warning by this (but only for typescrpt v2.6+).

Reply:
Also, isn't `@ts-ignore` only for `.js` files?

Reply:
no. it is for ts  only typescript 2.6+ ,  er.. ionic lock ts version at 2.4.2 now.
1. only use @ts-ignore without definition of interface Item under typescrpt 2.6+
2. declare interface Item without @ts-ignore usually (under typescript 2.4.2)